paasta_tools/cli/cmds/remote_run.py

```diff
@@ -35,7 +35,32 @@
 from paasta_tools.utils import SystemPaastaConfig
 
 
-def add_remote_run_args(parser):
+def start_args(parser):
```

Comment:
maybe ``apply_start_args_to_parser`` to make it clear that this function has side effects?